frappe/email/__init__.py

```diff
@@ -8,30 +8,15 @@ def sendmail_to_system_managers(subject, content):
 	frappe.sendmail(recipients=get_system_managers(), subject=subject, content=content)
 
 @frappe.whitelist()
-def get_contact_list(txt):
+def get_contact_list():
```

Comment:
why remove filter?

Reply:
Earlier based on input and change in input frappe.call was made and txt was matched with contacts and the matching contact was returned. Instead the whole contact list is fetched and then based on input the matching contact is filtered from the list.